### PR TITLE
Checkout success fixes

### PR DESCRIPTION
**Pull request by Jade-GG**
Address card:
- Allow custom titles
- Allow street to be a string (e.g. as retrieved from the `sales_order_address` table)

Checkout success:
- Grab addresses from order instead of from checkout component
- If there is more than 1 shipping addresses, assume the first one is a pickup location
- Format appropriately, clean up template with loop

### CODE REVIEW

File: resources/js/components/CheckoutSuccessAddresses.vue
Comment (royduin) [inline resources/js/components/CheckoutSuccessAddresses.vue:1]: We already have a checkout success component, maybe that's a better place for this? How is this handled within the new frontend? On that success page this info is also presented.

Comment (Jade-GG) [reply resources/js/components/CheckoutSuccessAddresses.vue:1]: This is true, but I would rather not overwrite any js in this package. Might be my own opinion here, I feel like that's something that could get very ugly very quickly.

With new frontend, do you mean the improve-frontend branch? That still uses the original cart data hack which is very not nice.

Comment (royduin) [reply resources/js/components/CheckoutSuccessAddresses.vue:1]: Maybe add this in the core within the improved frontend as the success page is also redone there.

File: resources/js/components/AddressCard.vue [hunk 1]
```diff
@@ -4,24 +4,29 @@
             address: Object,
             shipping: Boolean,
             billing: Boolean,
+            customTitle: {
```
Comment (indykoning) [inline resources/js/components/AddressCard.vue:7]: Is customTitle only used to be passed through this component?
Wouldn't it be better to handle this in the slot itself?

If it's a php variable we can immediately render it, and if it's a js or parent vue component variable we can work with that as well

**Review by indykoning — APPROVED** [review]

**Comment by royduin** [comment]
What about my comment? https://github.com/rapidez/checkout-theme/pull/28#discussion_r1305453363

**Comment by Jade-GG** [comment]
> What about my comment? [#28 (comment)](https://github.com/rapidez/checkout-theme/pull/28#discussion_r1305453363)

We've made a story to put it into the improve-frontend branch :)

**Pull request merged**